client/src/css/Aaron.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');
+
+h1,
+p,
+Table {
+  font-family: 'Inter', sans-serif;
+  margin: 0 auto;
+  text-align: center;
+}
+
+h1 {
+  padding-top: 15vh;
+  font-size: 60;
+}
+
+body {
+  background-color: #daad86;
+}
+.animated {
+  animation: shapeIn 1s;
+}
+@keyframes shapeIn {
+  from {
+    opacity: 0;
+    transform: translateY(-20px);
+  }
+  to {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+hr {
+  border: 1px solid black;
+  width: 30%;
+  margin-left: 3vh;
+  margin-bottom: -1vh;
+}
+
+.main-heading {
+  font-family: 'Quicksand', sans-serif;
+  font-weight: 600;
+  font-size: 300%;
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 4vh;
```

Comment:
use `vw` for width (so `margin-left` and `margin-right`) , `vh` for height (so `margin-bottom` and `margin-top`). check out [this resource](https://www.w3schools.com/cssref/css_units.asp).